test/karma.datagrid.tests.js

```diff
@@ -6,6 +6,10 @@ import { createXMochaHooks } from './utils/mochaHooks';
 
 const mochaHooks = createXMochaHooks();
 
+mochaHooks.beforeAll.forEach((mochaHook) => {
```

Comment:
This is to ensure the `config.disabled` flag is set to align with unit tests.
P.S. Karma does not support a `beforeAll` hook.

Reply:
Shouldn't we use beforeEach? What if a test forgets to restore it, or throws halfway?

Reply:
Overoptimization. 🙈 
I initially put it in `beforeAll` to avoid calling for every test. 🤔 
Thanks for the suggestion, I've moved it to `beforeEach` given that the cost to run it is negligible. 👌